src/bridge/command.rs

```diff
@@ -81,7 +79,7 @@ fn build_login_cmd_args(command: &str, args: &[&str]) -> (String, Vec<String>) {
     // -l: Skips changing directory to $HOME and prepending '-' to argv[0].
     // -p: Preserves the environment.
     // -q: Forces quiet logins, as if a .hushlogin is present.
-    let cmd_args = vec!["-flpq", &user, &shell, "-lc", &exec];
+    let cmd_args = vec!["-flpq", &user, "/bin/zsh", "-lc", &exec];
```

Comment:
This runs zsh as a login shell, which it shouldn't do, the alacritty code:
`login_command.args(["-flp", user, "/bin/zsh", "-fc", &exec])`

Reply:
The `-` to the exec ensures that the user shell runs in login mode

Reply:
I guess now we are cool

Reply:
I still see an extra `-l`, you have `-lc`, while alacritty has `-fc`

`-f` means `The shell shall disable pathname expansion`, I'm not sure if it's needed or not

Reply:
done